spec/yaml/swiftnav/sbp/linux.yaml

```diff
@@ -23,6 +23,7 @@ definitions:
  - MSG_LINUX_CPU_STATE:
     id: 0x7F00
     short_desc: List CPU state on the system
+    public: false
```

Comment:
One last tweak... there seems to be some cognitive dissonance between the meaning of "public" at the top level of the module and here on an individual message.  For the top level it just puts the message in a different location in the docs, for the message level it removes it from the docs.  To resolve this I would suggest we rename this to "hidden" and invert the logic, so `public: false` would become `hidden: true`.

Reply:
that is a good idea, but I think I'd like to do a separate PR that does that all at once if this looks good as is.

Reply:
Ok, logged a new ticket to track: https://swift-nav.atlassian.net/browse/INFRA-281

Reply:
I am thinking "publish:" (same logic) or "hide_from_docs:" (opposite logic).